Missing main entry in dist folder Version 3.0.2

Thank you for updating this package. I noticed that the index .js is missing in the dist folder for version 3.0.2. Looks like the same issue as #14 

`Cannot find module './node_modules/webpack-config-dump-plugin/dist/index.js'. Please verify that the package.json has a valid "main" entry`

Could you verify this and fix the release? 

Thank you!

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "webpack-config-dump-plugin",
   "description": "A webpack plugin to cache compiled webpack config on the file system",
-  "version": "3.0.2",
+  "version": "3.0.3",
   "author": "Sergey Nikitin <srg.post@gmail.com>",
   "license": "MIT",
   "main": "dist/index.js",
@@ -40,7 +40,7 @@
   },
   "scripts": {
     "format": "prettier --write .",
-    "build": "tsc",
+    "build": "tsc -p tsconfig.build.json",
     "test": "jest",
     "coverage": "jest --collect-coverage",
     "test:watch": "jest --watch --silent=false",

--- a/tsconfig.build.json
+++ b/tsconfig.build.json
@@ -1,0 +1,4 @@
+{
+  "extends": "./tsconfig",
+  "exclude": ["**/*.spec.ts", "**/__mocks__/*"]
+}

